src/main/java/fr/moribus/imageonmap/ui/SplatterMapManager.java

```diff
@@ -65,7 +65,8 @@
 import org.bukkit.inventory.ItemStack;
 import org.bukkit.inventory.meta.MapMeta;
 
-
+//TODO rework splatter effect, using ID is far more stable than nbt tags.
```

Comment:
For a future update as discussed using map IDs should be more stable than using nbt tags that often breaks with updates.